kmerexpr/exp_grad_solver.py

```diff
@@ -96,22 +110,26 @@ def exp_grad_solver(loss_grad,  x_0, lrs=None, tol=10**(-8.0), gtol = 10**(-8.0)
                 lrst = lrs[iter]
             # Update using prod exp function
             x_new = prod_exp_normalize(x, lrst*grad)
-        # x_new = softmax(np.log(x) +lrst*grad)  # 
+        # x_new = softmax(np.log(x) +lrst*grad)  #
```

Comment:
If this is intended as doc, move it up above with the Update doc.  If it's not doc, remove.